test/src/SpendPermissions/approve.t.sol

```diff
@@ -18,11 +19,12 @@ contract ApproveTest is SpendPermissionManagerBase {
         uint48 end,
         uint48 period,
         uint160 allowance,
-        uint256 salt
+        uint256 salt,
+        bytes memory extraData
     ) public {
         vm.assume(sender != address(0));
         vm.assume(sender != account);
-
+        console2.log(extraData.length);
```

Comment:
do we need to keep this?

Reply:
ah - no.  i take that out in the followup PR but will do so here too